ISISComputingGroup/EPICS-ioc
# Ticket983 tdk lambda genesys

ISISComputingGroup/IBEX#983

To Test:
Ensure that your submdules are updated and that there is is a TDK_LAMBDA_GENESYS listed in there
Re-make your whole instrument
Asking for a console with -x qualifier you should now see TDK_LAMBDA_GENESYS01 through to 04
Run any number of the IOCs you like, all should have records
Putting it into SIM should allow for the values through the current to field, and vice versa to update as appropriate

Note without a PSU to test with, this can only be simulated - I shall create some tickets for the final verification with an actual PSU
